tsup.config.ts

```diff
@@ -8,4 +8,5 @@ export default defineConfig({
   format: ['cjs', 'esm'],
   dts: true,
   outDir: 'dist',
+  noExternal: ['p-retry'],
```

Comment:
note: please make sure to retain the original license of `p-retry` in the distributable. We are legally obliged to do so.

Read https://github.com/sindresorhus/p-retry/blob/main/license for further details.